lang/java/reef-utils/src/main/java/org/apache/reef/util/MultiAsyncToSync.java

```diff
@@ -71,7 +71,8 @@ public MultiAsyncToSync(final long timeoutPeriod, final TimeUnit timeoutUnits) {
    * @throws Exception The future task object referenced by the {@code asyncProcessor}
    *                   parameter threw an exception.
    */
-  public <TAsync> boolean block(final long identifier, final FutureTask<TAsync> asyncProcessor) throws Exception {
+  public <TAsync> boolean block(final long identifier, final FutureTask<TAsync> asyncProcessor)
+        throws InterruptedException, InvalidIdentifierException, Exception {
```

Comment:
no need to declare generic `Exception` in `throws` anymore

Reply:
Done